### PR TITLE
Adds a jobs feature to make rust_g.dm work again

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
In #51 I foolishly assumed `jobs` was a feature in line with everything else, turns out it wasn't and now the jobs defines are missing from rust_g.dm.

~~I wonder if feature-interdependencies are possible...~~

They are. `jobs` is now a feature depended on by `http` and `sql`.

### CODE REVIEW

**Review by Cyberboss — APPROVED** [review]

**Comment by vuonojenmustaturska** [comment]
oh yeah this is tested to compile with default features and `--features="dmi git log json noise" --no-default-features`, `jobs` is not directly a default feature but a dependency of default features `http` and `sql`

**Review by Cyberboss — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ lto = true
 
 [dependencies]
 thiserror = "1.0"
-flume = "0.9"
+flume = {version = "0.9", optional = true }
```
Comment (Cyberboss) [inline Cargo.toml:20]:
```suggestion
flume = { version = "0.9", optional = true }
```
_eye twitch_

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Comment by SpaceManiac** [comment]
hmm today I will click "Rebase Merge"

**Comment by Cyberboss** [comment]
Why is it the default then?

**Comment by SpaceManiac** [comment]
it remembers what you used last

**Pull request merged**